worker/src/handlers/schedule.ts

```diff
@@ -22,9 +23,19 @@ export async function handleSchedule(event: ScheduledEvent): Promise<void> {
     currentDate.hour
   );
   const timestampString = String(currentDateObj.getTime());
-  storedAnalytics[timestampString] = currentDataset;
+  const storeKey = `history:${timestampString}`;
 
-  await KV.put("core_analytics", JSON.stringify(storedAnalytics));
+  for (const key of Object.keys(storedAnalytics)) {
+    core_analytics[key] = {
+      active_installations: storedAnalytics[key].active_installations,
+      installation_types: storedAnalytics[key].installation_types,
+    };
+  }
```

Comment:
Wouldn't it make more sense to turn this around (would then ofc also need site change):
```
active_installations: [{timestamp: number}, {timestamp: number}, {timestamp: number}, {timestamp: number}]
```

Reply:
For a next PR, we should probably also send less old data, no need to send hourly data for a year back, that can just be daily?

Reply:
Want to eventually have years, months/weeks, days, and hours depending on how far back the date is.

Reply:
> Wouldn't it make more sense to turn this around

_Maybe_ but the current structure is more true to the original datasets.